index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+<body class="background">
+    <header class="navbar">
+        <div class="logo">
+          <a href="#">MyLogo</a>
+        </div>
+        <div>
+          <a href="#">
+            <i class="fa fa-bars"></i>
+          </a>
+        </div>
+    </header>
+    <section class="onscreen">
+      <h1 class="bold">I'm Amanulla </h1>
+      <h1 class="bold">Glad to see you!</h1>
```

Comment:
Please stick to Figma design:

Your:
<img width="282" alt="image" src="https://user-images.githubusercontent.com/7243132/170593391-78801fd5-bc39-4b73-8066-59ab348511f5.png">

Figma:
<img width="282" alt="image" src="https://user-images.githubusercontent.com/7243132/170593423-b5ee2b6d-6135-4007-870d-8c51be6c8b92.png">